atmo/jobs/admin.py

```diff
@@ -3,6 +3,27 @@
 from .models import SparkJob
 
 
+def run_now(modeladmin, request, queryset):
+    for job in queryset:
+        job.run()
+
+
 @admin.register(SparkJob)
 class SparkJobAdmin(admin.ModelAdmin):
-    pass
+    list_display = [
+        'identifier',
+        'size',
+        'created_by',
+        'start_date',
+        'end_date',
+        'last_run_date',
+        'is_enabled',
+        'most_recent_status',
+    ]
+    list_filter = [
```

Comment:
Mind adding `start_date`, `end_date`, `last_run_date`, `interval_in_hours` to `list_filter`?